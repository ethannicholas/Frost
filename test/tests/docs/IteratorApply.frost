method process(s:String) {
    Console.printLine(s.length)
}

method main() {
    def words := "This will do something with the words in this string".find(/\w+/)
    words.map(m => m.groups[0]).apply(process)
}
