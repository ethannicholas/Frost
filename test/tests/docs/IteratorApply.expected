4
4
2
9
4
3
5
2
4
6
